Containerfile

```diff
@@ -1,17 +1,52 @@
-FROM registry.access.redhat.com/ubi9/ubi-minimal AS build
+FROM registry.fedoraproject.org/fedora:latest AS build_python
 
-RUN \
-    microdnf install -y \
+RUN dnf install -y \
         python3-pip \
         python3-pyyaml \
-        && microdnf clean all
+        && dnf clean all
 
 WORKDIR /src
 
 COPY pyproject.toml /src/
 COPY src /src/src
 
 RUN pip install --no-dependencies .
-RUN rm -rf src/*
+
+# ----
+FROM registry.access.redhat.com/ubi9/go-toolset:latest AS build_go
+
+ARG CONTAINERS_STORAGE_THIN_TAGS="containers_image_openpgp exclude_graphdriver_btrfs exclude_graphdriver_devicemapper"
+ENV CONTAINERS_STORAGE_THIN_TAGS=${CONTAINERS_STORAGE_THIN_TAGS}
+
+ARG IMAGES_REF="github.com/osbuild/images"
+ENV IMAGES_REF=${IMAGES_REF}
+
+# Note that "external" will most likely in the future build from internal
+# sources instead of pulling off the network
+RUN sh -c 'for otk_cmd in gen-partition-table \
```

Comment:
I will risk another premature comment here but I think we want to use the Makefile and not duplicate this list :)

Reply:
sorry - my mistake - this commit should not be here